spec/genre_spec.rb

```diff
@@ -0,0 +1,37 @@
+require_relative '../lib/genre'
+require_relative '../lib/item'
+describe Genre do
+  before :all do
```

Comment:
[OPTIONAL]

Better to keep one blank line after require lines to keep the it distinct from actual code.